packages/webdriverio/src/commands/browser/uploadFile.js

```diff
@@ -0,0 +1,47 @@
+/**
+ *
+ * Uploads a file to the selenium server by using the [`file`](/api/protocol/file.html) command. Note that
+ * this command might not be supported as it is an undocumented Selenium command.
+ *
+ * @alias browser.uploadFile
+ * @param {String} localPath local path to file
+ * @type utility
+ * @uses protocol/file
+ *
+ */
+import fs from 'fs'
+import path from 'path'
+import archiver from 'archiver'
+
+export default async function uploadFile (localPath) {
+    /**
+     * parameter check
```

Comment:
I'm not sure that it is useful comment

Reply:
it is because the `fs.createReadStream` would choke with a undescriptive error